frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -505,6 +505,34 @@ common_settings.document = {
     -- submenus are filled by menu_order
 }
 
+local metadata_folder_str = {
+    ["doc"] = _("book folder"),
+    ["dir"] = "koreader/docsettings",
+}
+
+local function genMetadataFolderMenuItem(value)
+    return {
+        text = metadata_folder_str[value],
+        checked_func = function()
+            return G_reader_settings:readSetting("document_metadata_folder") == value
+        end,
+        callback = function()
+            G_reader_settings:saveSetting("document_metadata_folder", value)
+        end,
+    }
+end
+
+common_settings.document_metadata_folder = {
+    text_func = function()
+        local value = G_reader_settings:readSetting("document_metadata_folder", "doc")
+        return T(_("Book metadata folder: %1"), metadata_folder_str[value])
+    end,
```

Comment:
I think I'd rather see "Book settings folder", document_settings_folder,  used instead of "metadata" all around there, it feels less ambiguous than book metadata (ie. author, title).
Unless it's confusing in other ways to use "settings" in the code .

Reply:
But may be it expresses more clearly where the `"metadata".epub.lua` will be kept ? :)